app/experimenter/nimbus-ui/schema.graphql

```diff
@@ -177,12 +177,17 @@ type NimbusFeatureConfigType {
   nimbusexperimentSet: [NimbusExperimentType!]!
 }
 
+enum NimbusIsolationGroupRandomizationUnit {
```

Comment:
Why isolationgroup?

Reply:
Why not isolation group? 🤔 

Also becuase randomization unit is a property of isolation group:

https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/models/nimbus.py#L212